lib/opentelemetry/api/utils/logger/base.lua

```diff
@@ -0,0 +1,114 @@
+------------------------------------------------------------------------------------------------------------------------
+--- A simple logger that uses print(). All loggers must adhere to the same interface.
+--
+-- @module api.utils.logger
+------------------------------------------------------------------------------------------------------------------------
+local _M = {}
+
+function _M.module_name()
+    return "api.utils.logger.base"
+end
+
+--- Return a table with all log levels.
+--
+-- @return A table of log levels ({ log_level_name = int, ... }).
+function _M:log_levels()
+    return { debug = 8, info = 7, notice = 6, warn = 5, error = 4, crit = 3, alert = 2, emerg = 1 }
+end
+
+--- Return a new logger instance
+--
+-- @string log_level The log level to use. Defaults to 'error'.
+-- @return A new logger instance
+function _M:new(log_level)
+    return setmetatable(
+        { log_level = self:log_levels()[log_level] or self:log_levels().error },
+        { __index = self })
+end
+
+--- Write message to stdout
+--
+-- @string message The message to write.
+-- @param[type=number] configured_level The log level of the logger instance.
+-- @param[type=number] callsite_level The log level at which the message was logged.
+--
+-- @return nil
+function _M:write(message, configured_level, callsite_level)
+    if configured_level >= self:log_levels()[callsite_level] then
```

Comment:
Can we just use `self.log_level` here, instead of `configured_level `?

Reply:
sorry for the delay on this, I will try to make this change!